### PR TITLE
Make *_qc variables optional

### PR DESCRIPTION
**Pull request by kwilcox**
I believe as of V3 that these `*_qc` variables are optional. I'm about to start uploading new files to the GliderDAC without the `*_qc` variables and the CC check was failing due to the missing variables.

Is this correct? Should we make a `gliderdac_v2` and `gliderdac_v3` checker to make these checks backwards compatible?

### CODE REVIEW

**Comment by rsignell-usgs** [comment]
Shouldn't there always be `_qc` variables, but with a flag value of `no_quality_checks`?

**Comment by kwilcox** [comment]
This wasn't meant to be a discussion on the GliderDAC file format. I think that discussion belongs here: https://github.com/ioos/ioosngdac/issues

**Comment by kwilcox** [comment]
py3.4 checks are failing with a oldie-but-goodie conda-forge issue. Thumbs up if you are in favor of dropping 3.4 tests and adding 3.6 tests.

**Comment by rsignell-usgs** [comment]
@kwilcox is there an emoji for face-being-slapped?

**Comment by kwilcox** [comment]
:man_facepalming:

**Comment by rsignell-usgs** [comment]
🤕

**Comment by kwilcox** [comment]
**tl;dr** GliderDAC is capable of running QARTOD checks now so they don't need the provider to run the quality checks before uploading.

**Comment by lukecampbell** [comment]
@Bobfrat should take a look though. I'm no longer maintaining this project.
